cli/cmd_suggestion.go

```diff
@@ -79,11 +79,11 @@ func printSuggestions(inputs []string, root *cmds.Command) (err error) {
 
 	suggestions := suggestUnknownCmd(inputs, root)
 	if len(suggestions) > 1 {
-		err = fmt.Errorf("Unknown Command \"%s\"\n\nDid you mean any of these?\n\n\t%s", inputs[0], strings.Join(suggestions, "\n\t"))
+		err = fmt.Errorf("unknown Command \"%s\"\n\nDid you mean any of these?\n\n\t%s", inputs[0], strings.Join(suggestions, "\n\t"))
 	} else if len(suggestions) > 0 {
-		err = fmt.Errorf("Unknown Command \"%s\"\n\nDid you mean this?\n\n\t%s", inputs[0], suggestions[0])
+		err = fmt.Errorf("unknown Command \"%s\"\n\nDid you mean this?\n\n\t%s", inputs[0], suggestions[0])
 	} else {
-		err = fmt.Errorf("Unknown Command \"%s\"\n", inputs[0])
```

Comment:
This is going to alter some formatting in the output of go-ipfs. However I am not sure we should do formatting like this. Ideas?

Reply:
I'd leave these as-is and add a `//nolint` comment. These are really just user-facing error messages.